portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -103,26 +105,27 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     long timestamp = System.currentTimeMillis();
 
     if (student) {
-      populateDataStore(datastore, name, "student", msg, timestamp);
+      populateDataStore(datastore, name, "student", msg, timestamp, location);
```

Comment:
since all these calls are the same, maybe just set the string type with an if statement, and at the end just have one call to populateDataStore. That way there is just one place that needs to be updated if it needs to be changed again in the future.

Reply:
Oh yeah, that would be a lot simpler. Just fixed!